### PR TITLE
ui#1774 - Me State - Session Storage

### PR DESCRIPTION
**Pull request by qijieye**

### CODE REVIEW

File: Client/Auth/index.ts [hunk 1]
```diff
@@ -1,50 +1,39 @@
 import * as model from "../../model"
-import { PaxpayFeature } from "../../model/PaxpayFeature"
 import { Connection } from "../Connection"
+import { Session } from "./Session"
 
 export class Auth {
 	#roles?: string[]
-	#features?: PaxpayFeature[]
+	#features?: model.PaxpayFeature[]
 	#data: Partial<model.LoginResponse>
```
Comment (lucy-meachem) [inline Client/Auth/index.ts]: if this is in the session, why leave it duplicated here?

File: Client/Connection.ts [hunk 2]
```diff
@@ -63,11 +64,11 @@ export class Connection {
...
-								if (Array.isArray(param[1]))
-									return `${param[0]}=${param[1].join(",")}`
-								else if (param[1] != undefined)
-									return param.join("=")
+							.map(([key, value]) => {
+								if (Array.isArray(value))
+									return `${key}=${value.join(",")}`
+								else if (value != undefined)
+									return `${key}=${value}`
 								return undefined
```
Comment (Magnus93) [inline Client/Connection.ts]: Suggestion
```ts
.map(([key, value]) => 
 Array.isArray(value) ?
  : `${key}=${value.join(",")}`
  ? (value != undefined)
  : `${key}=${value}`
  ? undefined
)
```

File: Client/Auth/index.ts (hunk 1 above)
Comment (qijieye) [reply Client/Auth/index.ts]: right

File: Client/Connection.ts (hunk 2 above)
Comment (lucy-meachem) [reply Client/Connection.ts]: suggestion: no

**Review by Magnus93 — APPROVED** [review]

**Pull request merged**